### PR TITLE
New version: Sogou.SogouInput version 14.3.0.9006 (14.3a)

### PR DESCRIPTION
**Pull request by SpecterShell**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in Run [#3757](https://github.com/SpecterShell/Dumplings/actions/runs/8328913342).
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145088)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Error occurred while downloading installer from https://ime.sogoucdn.com/sogou_pinyin_14.3.0.9006.exe. Forbidden (403). (0x80190193).
 Error information: 
 Http Response: Forbidden
 Http Reason Phrase: Forbidden
 
 3/18/2024 3:22:11 PM

(Automated response - build 870.)

**Comment by stephengillie** [comment]
URL: https://ime.sogoucdn.com/sogou_pinyin_13.13.0.8820.exe 
Status Code: 200

(Automated message - build 870)

File: manifests/s/Sogou/SogouInput/14.3.0.9006/Sogou.SogouInput.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Protocols:
+- sgreg
+FileExtensions:
+- scel
+- scpf
+- ssf
+ReleaseDate: 2024-03-18
+Installers:
+- Architecture: x86
+  InstallerUrl: https://ime.gtimg.com/pc/build/_sogou_pinyin_[Release]_14.3.0.9006_0.exe
```
Comment (stephengillie) [inline manifests/s/Sogou/SogouInput/14.3.0.9006/Sogou.SogouInput.installer.yaml:18]: Hi @SpecterShell,

The `PackageUrl` leads to a slightly different URL - but it seems like a slightly different version number in the name. 

```
  InstallerUrl: https://ime-sec.gtimg.com/202403191007/ce4ed40503b739a1366258780417098d/pc/sogou_pinyin_guanwang_14.3a.exe
```

I would suggest it, except that the page doesn't allow access to download the file:

image

Is there a better `PacakgeUrl`?

Comment (SpecterShell) [reply manifests/s/Sogou/SogouInput/14.3.0.9006/Sogou.SogouInput.installer.yaml:18]: Actually there is no permanent link on their website and the link on the homepage will be updated every time you refresh the page.

The method I was using to obtain a permanent link no longer works starting from last month. This `PackageUrl` comes from their update API and should be the best one I've found.

Comment (SpecterShell) [reply manifests/s/Sogou/SogouInput/14.3.0.9006/Sogou.SogouInput.installer.yaml:18]: @stephengillie

**Review by stephengillie — APPROVED** [review]

**Pull request merged**